data/mods/Magiclysm/Spells/attunements/Boreal_Mage.json

```diff
@@ -48,5 +48,25 @@
     "energy_source": "MANA",
     "damage_type": "cold",
     "max_level": 35
+  },
+  {
+    "id": "borealmage_auroral_guard",
+    "type": "SPELL",
+    "name": { "str": "Auroral Guard" },
+    "description": "Create sturdy shell what wrap your body and gain almost perfect defence.",
```

Comment:
```suggestion
    "description": "Create a sturdy shell that wraps your body, gaining a much better defense.",
```
It isn't a perfect defense; that would be invulnerable and not have any encumbrance...